crates/ruff/src/rules/flake8_pytest_style/rules/assertion.rs

```diff
@@ -184,25 +184,27 @@ pub(crate) fn unittest_assertion(
     match func {
         Expr::Attribute(ast::ExprAttribute { attr, .. }) => {
             if let Ok(unittest_assert) = UnittestAssert::try_from(attr.as_str()) {
-                // We're converting an expression to a statement, so avoid applying the fix if
-                // the assertion is part of a larger expression.
```

Comment:
(Every change here is mechanical: take the RHS of `let fixable =`, remove the assignment, and move the check into the `if checker.patch` block, which is then indented.)